lib/drivers/asanteDriver.js

```diff
@@ -727,41 +690,42 @@ module.exports = function (config) {
   var asanteDownloadRecords = function (recordtype, callback) {
     var cmd = requestRecord(recordtype, false);
     var retval = [];
-    console.log('requesting recordtypes ' + getDataRecordName(recordtype));
+    debug('Requesting recordtypes ' + getDataRecordName(recordtype));
     function iterate(err, result) {
       cfg.deviceComms.emitLog(true);
       if (err) {
         if (err == 'TIMEOUT' && nak_count < 10) {
-          console.log('NAK #', ++nak_count);
+          debug('NAK #', ++nak_count);
           var resend = resendRecord();
           return asanteCommandResponse(resend, iterate);
         }
-        console.log('error in iterate');
+        debug('Error in iterate!');
         callback(err, result);
       } else if (result.valid) {
         if (result.descriptor == DESCRIPTORS.RESPONSE_RECORD.value) {
           // process record
           retval.push(result.datarecord);
-          // console.log(result);
           // request next record
           var next = nextRecord();
-          if (retval.length >= 3000) {    // 3000 is bigger than any log's capacity
+          if (retval.length >= 3000) { // 3000 is bigger than any log's capacity
             next = stopSending();
-            console.log('cutting it short for debugging!');
+            // TODO: ask KJQ whether the cutting it short applies only to values
```

Comment:
Yeah, we talked about this, but then I got confused when I actually read the comment, so I added the note. Maybe I'll adjust the comments a bit for clarity.